### PR TITLE
cast float/int 64 to 32 in alibi-detect-server

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Closes https://github.com/SeldonIO/seldon-core/issues/3940

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by RafalSkolasinski** [comment]
performance drop from re-casting should be negligible  from majority of cases 

image

**Comment by axsaucedo** [comment]
Seems needs rebase as well for python and docs tests fix

File: components/alibi-detect-server/adserver/protocols/util.py [hunk 1]
```diff
@@ -35,3 +35,13 @@ def default(self, obj):  # pylint: disable=arguments-differ,method-hidden
         elif isinstance(obj, (np.ndarray,)):
             return obj.tolist()
         return json.JSONEncoder.default(self, obj)
+
+
+def read_inputs_as_numpy(inputs):
```
[the remark this answers is not in the document]
Comment (RafalSkolasinski) [reply components/alibi-detect-server/adserver/protocols/util.py]: added docstring

**Comment by axsaucedo** [comment]
Nice one - looks good
/appove

FYI @ascillitoe

**Pull request merged**